Project-2/README.md

```diff
@@ -10,42 +10,134 @@ Tip #4: Work together _as a team_.
 
 1. Let V be a vocabulary of ALCI consisting of a role name "P". Interpret part_of as "x is a part of y". Using this role name, define the following formulas in this language:
 ```
-  (a)  PP that says that x is a proper part of y
+  (a)  PP that says that x is a proper part of y 
+- P means x is a part of y
+- ~P means x is not a part of y
+- P^_ means y is a part of x
+  
+- A proper part is like a one-sided (subsumption) relationship.
+- Finn, Giacomo, Ali: PP:= ∃P.~P OR PP:= ∃(P⊓~P^_)
+
   (b)  iPP that says that y is a proper part of x
+ - Finn, Giacomo, Ali: iPP:= PP^_
+ 
   (c)  iP that says that y has x as part 
+iP := P^_
+
   (d)  O that says that x overlaps y
+ - Pulled from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1175958/: "r1 overlap r2 = [definition] for some r, r part_of r1 and r part_of r2."
+0 := ∃P.P^_
+ - Could ∃(P⊓P^_) also work for the above?
```

Comment:
This isn't grammatical, you need a role linking the quantifier to the content in the parentheses. With a role included you'd have all x such that some y = (x is part of y and y has x as part), but note that: 
P = P^_, that is, x part of y = y has x as part, so you've effectively said here all x such that some y = x is part of y.